data-prepper-plugins/elasticsearch/src/main/resources/otel-v1-apm-span-index-template.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": 1,
+  "version": 2,
```

Comment:
Look like since we added geo attribute, we need version bump ? Was this breaking anything ?

Reply:
I wasn't really sure about this, it would update unless I increased the version number. I suppose I could redownload the whole project instead of just swapping the .jar on my EC2.